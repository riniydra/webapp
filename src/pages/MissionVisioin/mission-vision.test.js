import { render } from "@testing-library/react";
import MissionVision from "./MissionVision";

describe("MissionVision", () => {
  it("renders correctly", () => {
    const tree = render(<MissionVision />);
    expect(tree).toMatchSnapshot();
  });
});
