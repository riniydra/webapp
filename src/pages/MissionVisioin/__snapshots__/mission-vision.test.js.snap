// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MissionVision renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="px-20 mt-6"
      >
        <div>
          <h1
            class="text-2xl font-semibold text-center"
          >
            Misson
          </h1>
          <div
            class="flex"
          >
            <div>
              <img
                alt="mission"
                class="w-[300rem] h-80 mt-5"
                src="mission.png"
              />
            </div>
            <p
              class="mt-5 text-lg px-5"
            >
              To empower individuals and communities worldwide by facilitating mutual assistance and support through Saayam, a software platform that connects requestors, volunteers, volunteer organizations, and donors. Our mission is to provide timely and targeted assistance to those facing challenges in their lives, fostering a culture of compassion, solidarity, and service to humanity.
            </p>
          </div>
        </div>
        <div>
          <h1
            class="text-2xl font-semibold text-center mt-6"
          >
            Vision
          </h1>
          <div
            class="flex items-start"
          >
            <p
              class="mt-5 text-lg px-5"
            >
              Saayam envisions a world where everyone has access to the help and resources they need to overcome obstacles and lead fulfilling lives. We aspire to be a global leader in humanitarian aid, leveraging technology and collaboration to create a seamless network of support that transcends geographical, cultural, and sociology-economic barriers. Our vision is to cultivate a community where the ethos of "Manava Sevaye Madhava Seva" (Service to mankind is Service to God) and "Sarve jana sukhino bhavantu" (May all live happily) guides our actions, driving positive change and fostering a more compassionate and equitable world for all.
            </p>
            <div>
              <img
                alt="vision"
                class="w-[350rem] h-80 mt-5"
                src="vision.png"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="px-20 mt-6"
    >
      <div>
        <h1
          class="text-2xl font-semibold text-center"
        >
          Misson
        </h1>
        <div
          class="flex"
        >
          <div>
            <img
              alt="mission"
              class="w-[300rem] h-80 mt-5"
              src="mission.png"
            />
          </div>
          <p
            class="mt-5 text-lg px-5"
          >
            To empower individuals and communities worldwide by facilitating mutual assistance and support through Saayam, a software platform that connects requestors, volunteers, volunteer organizations, and donors. Our mission is to provide timely and targeted assistance to those facing challenges in their lives, fostering a culture of compassion, solidarity, and service to humanity.
          </p>
        </div>
      </div>
      <div>
        <h1
          class="text-2xl font-semibold text-center mt-6"
        >
          Vision
        </h1>
        <div
          class="flex items-start"
        >
          <p
            class="mt-5 text-lg px-5"
          >
            Saayam envisions a world where everyone has access to the help and resources they need to overcome obstacles and lead fulfilling lives. We aspire to be a global leader in humanitarian aid, leveraging technology and collaboration to create a seamless network of support that transcends geographical, cultural, and sociology-economic barriers. Our vision is to cultivate a community where the ethos of "Manava Sevaye Madhava Seva" (Service to mankind is Service to God) and "Sarve jana sukhino bhavantu" (May all live happily) guides our actions, driving positive change and fostering a more compassionate and equitable world for all.
          </p>
          <div>
            <img
              alt="vision"
              class="w-[350rem] h-80 mt-5"
              src="vision.png"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
